src/arrayMethodSort.js

```diff
@@ -4,9 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = noArguments) {
+    let count = 0;
+
+    do {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const prev = this[i - 1];
+        const current = this[i];
+
+        if (compareFunction(prev, current) > 0) {
+          this[i - 1] = current;
+          this[i] = prev;
+          count++;
+        }
+      }
+    } while (count > 0);
+
+    return this;
   };
 }
 
+const noArguments = (a, b) => a.toString() > b.toString();
```

Comment:
The safest way is to use function declaration here.
And better call it something like `defaultSort`